### PR TITLE
Issue #457 - added ding_group and added link for menu or external link

### PR DESCRIPTION
**Pull request by bartvig**
Added ding_group as possible content type in entity reference.
Added field to use a link instead of entity reference and changed the view and template accordingly.

http://platform.dandigbib.org/issues/457

### CODE REVIEW

File: ding_tabroll.features.field_base.inc [hunk 1]
```diff
@@ -10,6 +10,37 @@
 function ding_tabroll_field_default_field_bases() {
   $field_bases = array();
 
+  // Exported field_base: 'field_ding_menu_ext'
+  $field_bases['field_ding_menu_ext'] = array(
```
Comment (cableman) [inline ding_tabroll.features.field_base.inc:14]: All fields should be prefix with the name of the module that defines them. Here it should be ding_tabroll not ding_menu!

File: ding_tabroll.info [hunk 2]
```diff
@@ -56,4 +58,4 @@ features[variable][] = scheduler_unpublish_required_ding_rolltab
 features[variable][] = scheduler_unpublish_revision_ding_rolltab
 features[variable][] = scheduler_use_vertical_tabs_ding_rolltab
 features[views_view][] = ding_tabroll
-mtime = 1444645542
+mtime = 1449053100
```
Comment (cableman) [inline ding_tabroll.info:61]: Please remove mtime from the info file.

**Comment by cableman** [comment]
Code looks good.

But the PR is against the old repository. Please re-do it with the new code from http://github.com/ding2/ding2 the code base have been consolidated there.

**Comment by guddo** [comment]
https://github.com/ding2/ding2/pull/104

You can close this PR.

**Pull request closed without merging**